ludwig/utils/triton_utils.py

```diff
@@ -49,142 +67,434 @@ def forward(self, {input_signature}):
-                    input_signature=_get_input_signature(config),
-                    input_dict=_get_input_dict(config),
-                    output_tuple=_get_output_tuple(config),
-                )
-            )
-
-        spec = importlib.util.spec_from_file_location("generated.ts", ts_path)
-        gen_ts = importlib.util.module_from_spec(spec)
-        spec.loader.exec_module(gen_ts)
-
-        gen_module = gen_ts.GeneratedInferenceModule(inference_module)
-        scripted_module = torch.jit.script(gen_module)
-    return scripted_module
+TRITON_ENSEMBLE_CONFIG_TEMPLATE = """name: "{model_name}"
+platform: "ensemble"
+max_batch_size: 0 # Dynamic batch not supported
+input [{input_spec}
+]
+output [{output_spec}
+]
+ensemble_scheduling {{
+  step [{ensemble_scheduling_steps}
+  ]
+}}
+"""
 
 
 def _get_type_map(dtype: str) -> str:
     """Return the Triton API type mapped to numpy type."""
     # see: https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md
     return {
-        "bool": "BOOL",
-        "uint8": "UINT8",
-        "uint16": "UINT16",
-        "uint32": "UINT32",
-        "uint64": "UINT64",
-        "int8": "INT8",
-        "int16": "INT16",
-        "int32": "INT32",
-        "int64": "INT64",
-        "float16": "FP16",
-        "float32": "FP32",
-        "float64": "FP64",
-        "string": "BYTES",
+        "bool": "TYPE_BOOL",
+        "uint8": "TYPE_UINT8",
+        "uint16": "TYPE_UINT16",
+        "uint32": "TYPE_UINT32",
+        "uint64": "TYPE_UINT64",
+        "int8": "TYPE_INT8",
+        "int16": "TYPE_INT16",
+        "int32": "TYPE_INT32",
+        "int64": "TYPE_INT64",
+        "float16": "TYPE_FP16",
+        "float32": "TYPE_FP32",
+        "float64": "TYPE_FP64",
+        "string": "TYPE_STRING",
+        "torch.float32": "TYPE_FP32",
+        "torch.float": "TYPE_FP32",
+        "torch.float64": "TYPE_FP64",
+        "torch.double": "TYPE_FP64",
+        "torch.float16": "TYPE_FP16",
+        "torch.half": "TYPE_FP16",
+        "torch.uint8": "TYPE_UINT8",
+        "torch.int8": "TYPE_INT8",
+        "torch.int16": "TYPE_INT16",
+        "torch.short": "TYPE_INT16",
+        "torch.int32": "TYPE_INT32",
+        "torch.int": "TYPE_INT32",
+        "torch.int64": "TYPE_INT64",
+        "torch.long": "TYPE_INT64",
+        "torch.bool": "TYPE_BOOL",
     }[dtype]
 
 
-def _get_input_spec(model: LudwigModel) -> str:
-    spec = []
-    for feature_name, feature in model.model.input_features.items():
-        metadata = model.training_set_metadata[feature_name]
-        spec.append(
-            TRITON_SPEC.format(
-                prefix="INPUT",
-                index=len(spec),
-                name=feature.feature_name,
-                data_type=_get_type_map(feature.get_preproc_input_dtype(metadata)),
-                data_dims=1,  # hard code for now
+def raw_feature_to_inference_input(s: pd.Series, feature) -> Union[List[str], torch.Tensor]:
```

Comment:
Can we use the `to_inference_model_input_from_series` in the `ludwig.utils.inference_utils.py` here?